src/otx/core/data/module.py

```diff
@@ -147,6 +149,8 @@ def train_dataloader(self) -> DataLoader:
             "pin_memory": True,
             "collate_fn": dataset.collate_fn,
             "persistent_workers": config.num_workers > 0,
+            "sampler": sampler,
+            "shuffle": sampler is None,
```

Comment:
Can we rely on data shuffling to custom sampler. I think some sampler wants shuffled dataset.
And also I cannot find shuffling in OTXSampler. Does this come from OTX1.x?

Reply:
https://github.com/pytorch/pytorch/blob/0dbef1618f19d69af075bc0c321f7c15c3595bfa/torch/utils/data/dataloader.py#L326
DataLoader does not normally allow shuffle and Sampler to coexist.